app/controllers/modules/course/module_host.rb

```diff
@@ -64,9 +78,46 @@ def get_settings_items(controller)
     class_methods_module.module_eval do
       include Sidebar
       include Settings
+      include Enableable
     end
   end
 
   # Eager load all the modules declared.
   eager_load_modules(File.join(__dir__, '../'))
+
+  # Initialize the module host instance
+  #
+  # @param [SettingsObject] instance settings object
```

Comment:
So '#settings' in this case ? This object is actually anything who can
handle settings

On Saturday, May 16, 2015, Joel Low <notifications@github.com> wrote:

> In app/controllers/modules/course/module_host.rb
> <https://github.com/Coursemology/coursemology2/pull/190#discussion_r30457315>
> :
>
> >      end
> >    end
> >
> >    # Eager load all the modules declared.
> >    eager_load_modules(File.join(__dir__, '../'))
> > +
> > +  # Initialize the module host instance
> > +  #
> > +  # @param [SettingsObject] instance settings object
>
> http://www.rubydoc.info/gems/yard/file/docs/Tags.md#Duck-Types
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/190/files#r30457315>.
>